### ISSUE TITLE
Pip version of Music is broken

### ISSUE THREAD

**Issue by malukus** [ttm/music#12]
I tried to install music in my Python, version 3.10, using pip install --upgrade music. However, it seems the only way I can get Python to import music is if I edit __init__.py to suppress some imports, otherwise it gives an import error talking about a circular import.
My Operating system is Windows 10, if that helps somehow.

**Comment by lakesare** [comment]
The import is failing for me in Google Colab, too.

<img width="998" alt="image" src="https://user-images.githubusercontent.com/7578559/161128644-ccb6a2aa-0a37-4763-9ee8-9a3a34a4122f.png">

**Comment by ttm** [comment]
should be solved with version 0.7b3 (published today).

**Comment by ttm** [comment]
@malukus @lakesare — closing the loop on this, four years late.

You were both right, and @malukus, your workaround diagnosed it exactly: commenting out imports in `__init__.py` was the only way through, because `music/__init__.py` imported `utils`, which imported `core`, which imported back into the still-initialising `music` package. A circular import that fired at import time, so it broke on every platform equally — Windows 10, and @lakesare's Colab notebook alike. Nothing about your environments was at fault.

**Fixed.** `import music` from a clean virtualenv is now part of the release check rather than something anyone hopes about, and CI runs the full suite on Python 3.10 through 3.13 for every push.

```console
pip install -U music
```

```python
import music
music.write_wav_mono(music.note(440, 1), "a.wav")
```

Verified from an empty virtualenv on 1.1.1, and it works in Colab. The package also has a [tutorial](https://ttm.github.io/music/tutorial.html) and an [API reference](https://ttm.github.io/music/) now, neither of which existed when you filed this.

Sorry it sat for four years with a one-line close.
